sycl/test/reduction/reduction_utils.hpp

```diff
@@ -52,3 +52,17 @@ struct CustomVecPlus {
     return CV(A.X + B.X, A.Y + B.Y);
   }
 };
+
+template <typename T>
+T getMinimumFPValue() {
+  return std::numeric_limits<T>::has_infinity
+             ? -std::numeric_limits<T>::infinity()
+             : std::numeric_limits<T>::lowest();
+}
+
+template <typename T>
+T getMaximumFPValue() {
+  return std::numeric_limits<T>::has_infinity
+             ? std::numeric_limits<T>::infinity()
+             : (std::numeric_limits<T>::max)();
```

Comment:
Smth funny with parentheses?

```suggestion
             : std::numeric_limits<T>::max();
```

Reply:
Hmm, that looked funny to me as well, but only until I found out that that is a common problem for min/max that give compilation error without those parentheses when 'windows.h' is included